Backend/src/KCTest.API/Startup.cs

```diff
@@ -31,7 +32,7 @@ public Startup(IConfiguration configuration)
         public void ConfigureServices(IServiceCollection services)
         {
 
-            services.AddControllers();
+            services.AddControllers().AddFluentValidation();
```

Comment:
Is working only with this line?

Reply:
yep @nminaya

Reply:
Huh, interesting. Not sure how is working, but seems like #4 will not be needed.

Or maybe is working because your registering the validators 🤔
https://github.com/Jadhielv/KCTest/blob/4459ea42f85ed0e12137cec3c63fdf5b087d360f/Backend/src/KCTest.API/Startup.cs#L52-L53

Reply:
yep, you're right